validator/src/test/java/org/odftoolkit/odfvalidator/InvalidPackageTest.java

```diff
@@ -145,44 +145,55 @@ public void validatePackage3() {
         output.contains("testInvalidPkg3.odt:  Info: 3 errors, no warnings"));
   }
 
-    @Test
-    public void validateEncryptedODT() {
-        String output = "";
-        try {
-            // password: hello
-            String name = "encrypted-with-pwd_hello.odt";
-            output = doValidation(name, null);
-        } catch (Exception t) {
-            StringWriter errors = new StringWriter();
-            t.printStackTrace(new PrintWriter(errors));
-            Assert.fail(t.toString() + "\n" + errors.toString());
-        }
-        Assert.assertTrue(output.contains("Fatal: only DEFLATED entries can have EXT descriptor"));
-        Assert.assertTrue(output.contains("Fatal: The document is encrypted. Validation of encrypted documents is not supported."));
-        java.util.logging.Logger.getLogger(getClass().getName()).log(Level.INFO, "Test result:\n{0}", output);
+  @Test
+  public void validateEncryptedODT() {
+    String output = "";
+    try {
+      // password: hello
+      String name = "encrypted-with-pwd_hello.odt";
+      output = doValidation(name, null);
+    } catch (Exception t) {
+      StringWriter errors = new StringWriter();
+      t.printStackTrace(new PrintWriter(errors));
+      Assert.fail(t.toString() + "\n" + errors.toString());
     }
+    Assert.assertTrue(output.contains("Fatal: only DEFLATED entries can have EXT descriptor"));
+    Assert.assertTrue(
+        output.contains(
+            "Fatal: The document is encrypted. Validation of encrypted documents is not supported."));
+    java.util.logging.Logger.getLogger(getClass().getName())
+        .log(Level.INFO, "Test result:\n{0}", output);
+  }
 
-    @Test
-    public void validateEncryptedODF13WithInvalidOpenPGP() {
-        String output = "";
-        try {
-            // password: hello
-            String name = "gpg13.odt";
-            output = doValidation(name, null);
-        java.util.logging.Logger.getLogger(getClass().getName()).log(Level.INFO, "Test result:\n{0}", output);
-        } catch (Exception t) {
-            StringWriter errors = new StringWriter();
-            t.printStackTrace(new PrintWriter(errors));
-            Assert.fail(t.toString() + "\n" + errors.toString());
-        }
-        Assert.assertTrue(output.contains("Fatal: only DEFLATED entries can have EXT descriptor"));
```

Comment:
this error indicates that the ZIP library has the bug referenced in https://github.com/tdf/odftoolkit/pull/48 i.e. it does not implement the current version of the ZIP specification.

switching to a different ZIP library only makes sense if it results in this error no longer happening.